ts/IdentificationModal.tsx

```diff
@@ -43,6 +50,10 @@ type State = {
   biometryType?: string;
 };
 
+type updateBiometrySupportPropType = {
```

Comment:
type names should start with a capital letter

Reply:
besides, is this type definition really needed? it contains only one property and it's used only in one place, perhaps just use a boolean?

Reply:
got rid of it.